src/agent/parser.cc

```diff
@@ -787,28 +787,29 @@ SpParser::FindFunction(string name,
 }
 
 // Find function by mangled name.
-bool
-SpParser::FindFunctionByMangledName(string name, FuncSet* found_funcs) {
+FuncSet
+SpParser::FindFunctionByMangledName(string name) {
 
-  sp_debug("agent", "LOOKING FOR FUNC BY MANGLED NAME - looking for %s",
+  sp_debug_agent("LOOKING FOR FUNC BY MANGLED NAME - looking for %s",
            name.c_str());
-  
+
+  FuncSet* found_funcs = new FuncSet();  
```

Comment:
This is a memory leak.  Nothing every frees this memory.  Change to `FuncSet found_funcs;` and `return found_funcs;`.